tests/testthat/test-verb-joins.R

```diff
@@ -447,8 +564,9 @@ test_that("join verbs generate expected ops", {
 })
 
 test_that("can optionally match NA values", {
-  lf <- lazy_frame(x = 1)
-  expect_snapshot(left_join(lf, lf, by = "x", na_matches = "na"))
+  lf1 <- lazy_frame(x = 1, .name = "lf1")
```

Comment:
Might be worth using `simulate_postgres()` here to get the slightly simpler SQL.